contracts/misc/UiPoolDataProviderV2V3.sol

```diff
@@ -111,9 +112,12 @@ contract UiPoolDataProviderV2V3 is IUiPoolDataProviderV3 {
 
       if (address(reserveData.underlyingAsset) == address(MKRAddress)) {
         bytes32 symbol = IERC20DetailedBytes(reserveData.underlyingAsset).symbol();
+        bytes32 name = IERC20DetailedBytes(reserveData.underlyingAsset).name();
```

Comment:
only these 3 lines changed - the rest is linting with the `pritter:write` config in the repo. Seems like it was not linted before.